src/mopi/run_training.py

```diff
@@ -54,14 +54,14 @@ def run_training(
             plugins=[OutputAnalyserPlugin()] + logger_plugins,
         )
 
-        store, pipeline = runner.train_test()
+        store, pipeline, unloaded_pipeline = runner.train_test()
 
         successes.append((experiment, pipeline, store))
 
-        if save_entire_pipeline is True:
+        if save_entire_pipeline is True and experiment.train is True:
             export_pipeline(
-                pipeline.id,
-                pipeline,
+                unloaded_pipeline.id,
+                unloaded_pipeline,
```

Comment:
We are now throwing back a pipeline on which the mutations happened (eg.: appending the configs of an experiment) but on which 
- the ``load`` functions weren't called
- the ``ids`` and ``parent_paths`` aren't appended (otherwise when it loads it appends it again and won't find the model)

Reply:
OK by me, right now, but will we ever be able to debug this in the future? the disparity between the pipeline we save vs the pipeline we run?
I'll forget about this in 3, 2, 1...
Is there an alternative? a train_test() method that doesn't do the overriding, etc, that we can call if the pipeline is "loaded from disk"?